conduct.md

```diff
@@ -101,7 +101,7 @@ Behavior that will lead to exclusion includes the following points, inspired by
 * Violence, threats of violence or violent language directed against another person.
 * Sexist, racist, homophobic, transphobic, ableist or otherwise discriminatory jokes and language.
 * Posting or displaying sexually explicit or violent material.
-* Posting or threatening to post other people’s personally identifying information ("doxing").
+* Posting or threatening to post other people’s personally identifying information (including so-called "doxing").
```

Comment:
I don't know if adding the word "including" clarifies here since as far as I know "Posting or threatening to post other people’s personally identifying information" perfectly describes "doxing".

Reply:
maybe it’s better to omit the parenthetical altogether and just not use the word “doxing” at all.